client/src/Components/Hobbies.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import '../Style/Hobbies.css'
+import HobbiesSlideshow from './HobbiesSlideshow'
+
+export default function Hobbies(props) {
+    return (
+        <div id="hobby-body">
+            <div id="words">
+                <br></br>What I Like...</div>
+            <br></br>
+            <div id="p2">
```

Comment:
These two `<div>` id's could be a little better. Something like "hobbies-words" or "hobbies-p2" would tell you what page it's on when you're referring to it through your css or query selectors